src/account/account.ts

```diff
@@ -26,7 +29,20 @@ export class Account implements IAccount {
     return act;
   }
 
+  public static fromAddressAndSigner(
```

Comment:
in many cases (probably not yours), the caller does not care about the target user's address. we may not need to provide address here at this level.

in the remote signer, there are two cases:

1. if no `from` address is provided, the signer can be whatever address the wallet has.
2. if `from` address is provided, the signer must be the `from` account in the wallet.